demisto_sdk/commands/lint/linter.py

```diff
@@ -993,8 +994,12 @@ def _docker_run_pwsh_analyze(self, test_image: str, keep_container: bool) -> Tup
     def _update_support_level(self):
         pack_dir = self._pack_abs_dir.parent if self._pack_abs_dir.parts[-1] == INTEGRATIONS_DIR else \
             self._pack_abs_dir.parent.parent
-        with (pack_dir / PACKS_PACK_META_FILE_NAME).open() as f:
-            pack_meta_content: Dict = json.load(f)
+        pack_metadata_file = pack_dir / PACKS_PACK_META_FILE_NAME
+        with FileLock(pack_metadata_file.as_posix()):
+            logger.debug(f'Lock acquired for {pack_metadata_file}')
```

Comment:
can we add more data as to which thread/function/whatever is the one that acquired it?

Reply:
True, added the package name in the debug message.
I don't think the function or the thread number will help us too much to debug it in the future.